recipes/cuda-api-wrappers/all/conanfile.py

```diff
@@ -44,6 +45,9 @@ def package(self):
     def package_info(self):
         self.cpp_info.bindirs = []
         self.cpp_info.libdirs = []
-
+        self.cpp_info.set_property("cmake_target_name", "cuda-api-wrappers::runtime-and-driver")
+        if Version(self.version) < "0.7.0":
+            # For previously published versions the target name was different, maintain compatibility
+            self.cpp_info.set_property("cmake_target_aliases", ["cuda-api-wrappers::cuda-api-wrappers"])
```

Comment:
For older versions and just a precaution I'm also adding the previously used target name - to avoid disruption if existing users update the recipe revision.

users wishing to move to a newer version will have to use the correct target name